http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/LeonardoService.scala

```diff
@@ -305,6 +321,18 @@ class LeonardoService(protected val dataprocConfig: DataprocConfig,
           dbRef.inTransactionIO { _.clusterQuery.updateClusterStatus(existingCluster.id, ClusterStatus.Updating, now) }.void
         } else IO.unit
 
+        //check update follow-up
```

Comment:
there is currently a bug in the below block, where if both clusterResized and shouldFollowup are true, an exception will occur as the cluster status will be 'Updating', and then the code will attempt to send it to "Stopping". This needs product resolution.